src/test/util/file_util/get_file_name_from_path_test.py

```diff
@@ -1,9 +1,10 @@
 # Copyright (c) 2020 Anastasiia Birillo, Elena Lyulina
 
+import pytest
```

Comment:
Pytest is not a standard library, so please, move this import into a separated second group